dolt_integrations/dvc/examples/fashion-mnist/dolt-dvc-demo.md

```diff
@@ -0,0 +1,551 @@
+incrememntally generate logs:
+```
+> > dolt sql -q "select * from summary"
++-----------+--------------+--------------------------------------+--------+
+| loss      | label_branch | timestamp                            | acc    |
++-----------+--------------+--------------------------------------+--------+
+| 1.6994121 | fashion      | 2021-04-02 10:29:12.840018 +0000 UTC | 0.5352 |
+| 1.580542  | master       | 2021-04-02 10:43:14.570946 +0000 UTC | 0.6217 |
++-----------+--------------+--------------------------------------+--------+
+```
+
+One instance where we might want to access the commit history is
+`dolt blame`’ing a data changes. For example, identifying who added
+certain rows to the fashion branch:
+```bash
+dolt blame fashion labels | head -n 5
++--------+----------------------------+-----------------+------------------------------+----------------------------------+
+| ROW_ID | COMMIT MSG                 | AUTHOR          | TIME                         | COMMIT                           |
++--------+----------------------------+-----------------+------------------------------+----------------------------------+
+| 14648  | Add Fashion labels         | Bojack Horseman | Wed Mar 31 16:20:50 PDT 2021 | 5fblpjp5neurvsfp989s6ea9lt01vd2q |
+| 27051  | Add Fashion labels         | Bojack Horseman | Wed Mar 31 16:20:50 PDT 2021 | 5fblpjp5neurvsfp989s6ea9lt01vd2q |
+```
+
+You can learn more about how to use Dolt’s Git and SQL features
+[here](https://docs.dolthub.com).
+
+## How Dolt Integrates With DVC
+
+### Adding
+
+DVC’s main metadata is the md5-hash used to sync local and remote
+filesystems. Usually, DVC hashes files and directories the same
+way as Git. To avoid duplicating what Dolt does internally, we
+substituted Dolt’s commits in-lieu of an md5 hash.
+
+```bash
+> cat data/labels.dvc
+outs:
+- md5: tkcao72e0upe2umb2kokartgivd9keqc-pinisl8m4tfdecb8iikqhicta61nou9s.dolt
+  size: 1465033
+  path: labels
+  cache: false
+```
+
+Dolt's three  heads tell us 1) the last commit, 2) the hash if
+all files in our working directory were committed now
+, 3) the ongoing index/staging commit hash.
+The first two, HEAD and working, let us monitor whether the
+database has changed since the most recent DVC-add. If necessary,
+the HEAD commit can be used to update to the appropriate database version.
+
+## Commits
+
+In DVC, output lineage is captured as Git-committed YAML files.
+Pre-defined output paths are saved as-is when a workflow completes.
+
+Dolt users are responsoble for committing changes.
+If a new database state is committed within a workflow,
+DVC will track the new commit.
+If a tracked database is changed but not committed by the end of a
+workflow, then we have an uncommitted transaction -- a state that
+Dolt cannot reproduce.
+```bash
+> dvc add
+Adding...
+ERROR: unexpected error - Dolt status is not clean; commit a reproducible state before adding.
+```
+
+Fixing this error requires committing changes.
```

Comment:
So how do I do that? Show us

This should probably be in the main tutorial, not hidden down here